src/test/java/Homework22.java

```diff
@@ -0,0 +1,37 @@
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.LoginPage;
+
+public class Homework22 extends BaseTest{
+
+    @Test
+    public void renamePlaylist() {
+        loginPage.loginCorrectMethod();
+        playlistPage.clickPlaylist();
+        playlistPage.rightCLickPlaylist();
+        playlistPage.clickEdit();
+        playlistPage.enterNewName();
```

Comment:
Please, add Assert

**One test - One Assert**

For example

![Clean_Test](https://github.com/acl26/A49-QA-Automation-Framework/assets/104761482/629681a3-8bd4-41e2-82de-a650614b3b2e)